geoportailv3/static/js/share/sharedirective.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @fileoverview This file provides a list of social sharing options.
+ * This directive is used
+ * to create a sharing panel in the page.
+ *
+ * Example:
+ *
+ * <app-share></app-share>
+ *
+ */
+goog.provide('app.shareDirective');
+
+goog.require('app');
+goog.require('goog.Uri');
+
+
+/**
+ * @param {string} appShareTemplateUrl Url to share template
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+app.shareDirective = function(appShareTemplateUrl) {
+  return {
+    restrict: 'E',
+    scope: {
+      'visible': '=appShareVisible'
+    },
+    controller: 'AppShareController',
+    controllerAs: 'ctrl',
+    bindToController: true,
+    templateUrl: appShareTemplateUrl
+  };
+};
+
+app.module.directive('appShare', app.shareDirective);
+
+
+
+/**
+ * @ngInject
+ * @constructor
+ * @param {angular.$window} $window
+ * @param {angularGettext.Catalog} gettextCatalog Gettext catalog.
+ * @export
+ */
+app.ShareDirectiveController = function($window, gettextCatalog) {
+  /**
+   * @type {Object}
+   */
+  this.services = {
+    'facebook': {'url': 'https://www.facebook.com/sharer/sharer.php'},
+    'twitter': {'url': 'https://www.twitter.com/intent/tweet'},
+    'googlePlus': {'url': 'https://plus.google.com/share'}
+  };
```

Comment:
Any reason for making this object public? And any reason for using quotes for the keys? Do you need to prevent renaming?